cmd/securetrigger/main.go

```diff
@@ -0,0 +1,40 @@
+package main
```

Comment:
Is this a correct location for this? Where do I put dockerfile for this? Maybe:
cmd/securetrigger/github/main.go  is better?

Reply:
Maybe `cmd/validate-github`, `cmd/validate-gitlab`

Reply:
I am changing to shell-based task as suggested by akihikokuroda.